src/main/kotlin/no/nav/familie/ef/mottak/task/TaskFlyt.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.familie.ef.mottak.task
+
+fun kafkaHendelseFlyt() = listOf(LagEksternJournalføringsoppgaveTask.TYPE, SjekkOmJournalpostHarFåttEnSak.TYPE)
+
+fun hovedflyt() = listOf(
+    LagPdfTask.TYPE,
+    ArkiverSøknadTask.TYPE,
+    LagBehandleSakOppgaveTask.TYPE,
+    OpprettSakTask.TYPE,
+    OppdaterBehandleSakOppgaveTask.TYPE,
+    OppdaterJournalføringTask.TYPE,
+    FerdigstillJournalføringTask.TYPE
+)
+
+fun fallbackFlyt() = listOf(LagJournalføringsoppgaveTask.TYPE, HentSaksnummerFraJoarkTask.TYPE)
+fun fallbackTask() = fallbackFlyt().first() // Akkurat nå er alle fallback-tasks det samme, men utvid logikk her dersom det skulle endres
+
+fun String.nesteHovedflytTask() = hovedflyt().zipWithNext().first {this == it.first}.second
+fun String.nesteFallbackTask() = fallbackFlyt().zipWithNext().first {this == it.first}.second
+fun String.nesteKafkaHendelseFlyt() = kafkaHendelseFlyt().zipWithNext().first { this == it.first }.second
```

Comment:
Fint å vise hovedflytet og hvordan hele det ser ut. 👍 
Er litt usikker på de andre metodene. Er det kanskje bare enklere å bruke riktig "fallback" direkt i de taskene? 

For meg virker det som att `LagBehandleSakOppgaveTask` tidligere hadde `OpprettSakTask.TYPE` som nå er `LagJournalføringsoppgaveTask.TYPE`

Blir litt vanskelig å følge flytet for fallback nå. Når burde man bruke `fallbackFlyt` vs `nesteFallbackTask`?
Hadde det vært enklere å ha en map som viser fallback? Eks
```
val fallbacks = mapOf(
        ArkiverSøknadTask.TYPE to LagJournalføringsoppgaveTask.TYPE
)
fun String.nesteFallbackTask() = fallbacks[this] ?: error("Finner ikke fallback til $this")
```

Reply:
Liker det litt, men føles også litt "skummelt" å utvide `String` med denne her 🙀  ?

Reply:
Det er kanskje like greit å opprette riktig task med en gang. Muligens overkill med fallbackflyt, ville bare vise at de to oppgavene hang sammen for det caset. Likte mappet for fallback, det funker :) Også er det selvfølgelig veldig vanskelig å være uenig i å ha flere tester for fallback-casene.

Ja, enig Charlie. Det plagde meg at Task var del av felles bibliotek, ellers ville jeg selvfølgelig lagt logikken der. Det blir vel ikke helt riktig å oppdatere Task i felles-repoet heller, siden dette er noe som kun trengs i mottak.

Hvis det ses på som en minimal forbedring, pluss at det ble litt dumt med jobbing på string, så er det vel kanskje like greit å droppe dette løsningsforslaget?

Reply:
> For meg virker det som att `LagBehandleSakOppgaveTask` tidligere hadde `OpprettSakTask.TYPE` som nå er `LagJournalføringsoppgaveTask.TYPE`

Det hørtes ikke helt bra ut, mulig jeg har oversett noe i fallbacken altså. Skal passe på å få fikset hvis dette løsningsforslaget ønskes.

Reply:
> > For meg virker det som att `LagBehandleSakOppgaveTask` tidligere hadde `OpprettSakTask.TYPE` som nå er `LagJournalføringsoppgaveTask.TYPE`
> 
> Det hørtes ikke helt bra ut, mulig jeg har oversett noe i fallbacken altså. Skal passe på å få fikset hvis dette løsningsforslaget ønskes.

Tok en titt på denne: Det er bare endret på == til != i nesteTask-sjekken. Det var for å være konsekvent med at hovedflyt skal være først, mens fallback blir i else-statement, så endringen betyr: OpprettSakTask skal opprettes dersom LagBehandleSak gikk bra. Dersom LagBehandleSak ikke gikk bra skal det nå opprettes LagJournalføringsoppgaveTask (som tidligere var HentSaksnummerFraJoarkTask).